flakeheaven/_version.py

```diff
@@ -1 +1,9 @@
-__version__ = '0.11.0'
+try:
+    # built-in
+    from importlib import metadata
+except ImportError:
+    # Running on pre-3.8 Python; use importlib-metadata package
+    # external
+    import importlib_metadata as metadata  # type: ignore
+
+__version__ = metadata.version("flakeheaven")
```

Comment:
poetry requires that we define the version in `pyproject.toml` instead of in the code [ref](https://python-poetry.org/docs/pyproject/#version)

this seemed like an acceptable way to avoid having to update the version in two places